core/src/main/java/org/apache/struts2/interceptor/DefaultResourceIsolationPolicy.java

```diff
@@ -0,0 +1,49 @@
+package org.apache.struts2.interceptor;
```

Comment:
You are missing a header with license, see other files for an example

Reply:
just pushed another commit adding the missing licenses